tools/PI/DevHome.PI/Styles/CommandBarStyle.xaml

```diff
@@ -0,0 +1,836 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.PI.Styles">
+
+    <!-- This is a copy/paste of the stock Command Bar template, with a small change to the "More Items" button -->
```

Comment:
Consider adding the link to the original code, and briefly elaborate on the change that was made